.github/labeler.yml

```diff
@@ -16,6 +17,12 @@ kind/build:
   - "docker/**"
   - "webpack.config.js"
 
+theme/package-registry:
+  - "modules/packages/**"
```

Comment:
and `services/packages`/`routers/api/packages`

Reply:
Yes, another PR